routers/web/explore/code.go

```diff
@@ -25,8 +25,13 @@ func Code(ctx *context.Context) {
 		ctx.Redirect(setting.AppSubURL + "/explore")
 		return
 	}
+	if setting.Service.Explore.DisableCodePage {
```

Comment:
This condition can be merged into the condition above?
Why especially redirect to `/explore/repos` not `/explore`, it should be handled by router?
If we added `DisableReposPage`, then it will redirect to a 404 page?

Reply:
b1b8e3e2afe4002bcc672c5905827bfc85b5031e